# HG changeset patch
# User Ken Moffat <zarniwhoop@users.sourceforge.net>
# Date 1193435051 -3600
# Node ID 8a369a0d5ff104cc115bbe76c6b4f3e281d1ef25
# Parent 05fde4048642d1a7edb19ea623acd760064718bc
Linux-2.6.23 header change breaks build
>
> /* Special codes used when specifying changer slots. */
> #define CDSL_NONE               (INT_MAX-1)
> #define CDSL_CURRENT            INT_MAX

--- a/src/input/vcd/libcdio/_cdio_linux.c	Tue Oct 23 23:45:58 2007 +0100
+++ b/src/input/vcd/libcdio/_cdio_linux.c	Fri Oct 26 22:44:11 2007 +0100
@@ -59,6 +59,7 @@ static const char _rcsid[] = "$Id: _cdio
 #include <unistd.h>
 #include <fcntl.h>
 #include <mntent.h>
+#include <limits.h>
 
 #include <linux/cdrom.h>
 #include <scsi/scsi.h>

